config/locales/en.yml

```diff
@@ -1489,16 +1489,43 @@ en:
           weir: "Weir"
           "yes": "Waterway"
       admin_levels:
-        level2: "Country Boundary"
-        level3: "Region Boundary"
-        level4: "State Boundary"
-        level5: "Region Boundary"
-        level6: "County Boundary"
-        level7: "Municipality Boundary"
-        level8: "City Boundary"
-        level9: "Village Boundary"
-        level10: "Suburb Boundary"
-        level11: "Neighbourhood Boundary"
+        level2: "International Boundary"
+        level3: "Administrative Boundary (Level 3)"
+        level4: "Administrative Boundary (Level 4)"
+        level5: "Administrative Boundary (Level 5)"
+        level6: "Administrative Boundary (Level 6)"
+        level7: "Administrative Boundary (Level 7)"
+        level8: "Administrative Boundary (Level 8)"
+        level9: "Administrative Boundary (Level 9)"
+        level10: "Administrative Boundary (Level 10)"
+        level11: "Administrative Boundary (Level 11)"
+      border_types:
+        arrondissement: "Arrondissement Boundary"
+        borough: "Borough Boundary"
+        cercle: "Cercle Boundary"
+        city: "City Boundary"
+        comarca: "Comarca Boundary"
+        county: "County Boundary"
+        departement: "Department Boundary"
+        department: "Department Boundary"
+        district: "District Boundary"
+        distrito: "District Boundary"
+        freguesia: "Freguesia Boundary"
+        local_authority: "Local Authority Boundary"
+        municipality: "Municipal Boundary"
+        municipi: "Municipal Boundary"
+        município: "Municipal Boundary"
+        nation: "International Boundary"
+        national: "International Boundary"
+        neighbourhood: "Neighborhood Boundary"
+        parish: "Parish Boundary"
+        province: "Provincial Boundary"
+        região: "Regional Boundary"
+        region: "Regional Boundary"
+        state: "State Boundary"
+        town: "Town Boundary"
+        township: "Township Boundary"
+        village: "Village Boundary"
```

Comment:
For starters, I added a translatable string for each of the 30 or so `border_type=*` values that occur most frequently with `boundary=administrative`. Only one translatable string is provided per `border_type=*` value globally. This will likely present a problem for some languages that would translate English words like “district”, “county”, or “town” differently depending on the country. I think we’ll need to add some more specific strings qualified by country code, as explained in https://github.com/openstreetmap/openstreetmap-website/issues/5450#issuecomment-2566049746, but it’ll require a bit more research to find out which values are important in which countries.